api/app/tests/apps/email_integrations/integrations/test_core.py

```diff
@@ -182,6 +190,46 @@ def test_links_environment_env_var_not_set(self):
                 self.assertIn('http://dummy_link', message.body)
                 self.assertIn('http://dummy_link', message.alternatives[0][0])
 
+    def test_create_status_change_notification_message_child_signal(self):
+        # Prepare signal with status change to `AFGEHANDELD`.
+        status = StatusFactory.create(_signal=self.child_signal, state=workflow.AFGEHANDELD,
+                                      text='Done.')
+        self.child_signal.status = status
+        self.child_signal.status.save()
+        feedback = FeedbackFactory.create(_signal=self.child_signal)
+
+        txt_message, html_message = core.create_status_change_notification_message(
+            self.child_signal, self.child_signal.status, feedback
+        )
+
+        self.assertIn(str(self.child_signal.id), txt_message)
+        self.assertIn(self.child_signal.text, txt_message)
+        self.assertIn(self.child_signal.status.text, txt_message)
+        self.assertIn('Op 10 oktober 2018 heeft u', txt_message)
+
+        self.assertIn(str(self.child_signal.id), html_message)
+        self.assertIn(self.child_signal.text, html_message)
+        self.assertIn(self.child_signal.status.text, html_message)
+        self.assertIn('Op 10 oktober 2018 heeft u', html_message)
+
+    def test_create_status_changed_in_behandeling_message(self):
+        # Prepare signal with status change to `ON_HOLD`.
```

Comment:
INGEPLAND?

Reply:
Renamed the templates to `signal_status_changed_ingepland.html` and `signal_status_changed_ingepland.txt` also the status has changed to `INGEPLAND` and the function name is also changed to `test_create_status_changed_ingepland_message`